src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -93,16 +93,16 @@ public static void ReportNonFatal(Exception exception)
                 exceptionObject: exception,
                 gatherEventDetails: faultUtility =>
                 {
-                    // add current process dump
-                    faultUtility.AddProcessDump(currentProcess.Id);
-
                     // add ServiceHub log files:
                     foreach (var path in CollectServiceHubLogFilePaths())
                     {
                         faultUtility.AddFile(path);
                     }
```

Comment:
should we also avoid adding these files?  my understanding is this is a lot of non-trivial IO whch also majrly slows things down.

Reply:
@tmat do we need these files for NFE?

Reply:
What you can do is check FaultUtility.IsIncludedInWatsonSample, and only collect those files if true.

Reply:
Yes, we need those files.